app/assets/javascripts/controllers/openshift_deployment/openshift_deployment_controller.js

```diff
@@ -0,0 +1,10 @@
+miqHttpInject(angular.module('openshiftDeployment', ['ui.bootstrap', 'patternfly', 'patternfly.charts', 'miq.card']))
```

Comment:
why is there a need for miq.card and patternfly.charts? is this ui going to look like a dashboard?
(while at it - is there a mockup/screenshot to understand how is this screen expected to look like?)